src/CoursePageResources/index.scss

```diff
@@ -0,0 +1,48 @@
+.container-fluid {
+    background-color: #EAF3FB;
+    color: #2E8ED3;
+}
+
+.container-fluid-container {
+    border-color: #B2D5EE;
+}
+
+.container-fluid-enabled-button {
+    border-color: #0075B4;
+    color: #2E8ED3;
+}
+
+.container-fluid-enabled-button:hover {
+    color: inherit;
+}
+
+.sub-container-text {
+    color: #39AAE3;
+}
+
+.border-enabled {
+    border-color: #4293C4 !important; 
+    border-width: 2px;
+    border-style: solid;
+    border-radius: 5px;
+}
+
+.border-disabled {
+    border-color: #ffffff !important;
+    border-width: 2px;
+    border-style: solid;
+    border-radius: 5px;
+}
+
+.container-fluid-button {
+    background-color: #0075B4;
+    color: #ffffff;
+    font-size: 12px;
+    &:hover {
+        color: #ffffff;
+    }
+}
```

Comment:
This is a very generic name for this button, and again uses the bootstrap names instead of something like `course-pages-button` or something. 
Other points:
- Colours should be in the theme, not here.
- Font size should be in rem not pixels.